lib/llvm/import-llvm

```diff
@@ -42,13 +45,18 @@ Support_imports = [
 Support_imports += [
     'COFF', 'ConvertUTF', 'ConvertUTFWrapper', 'Debug', 'FileSystem',
     'FileUtilities', 'Host', 'Locale', 'Path', 'Process', 'Program', 'SMLoc',
-    'SourceMgr', 'Signals', 'Unicode', 'UnicodeCharRanges']
+    'SourceMgr', 'Signals', 'Unicode', 'UnicodeCharRanges', 'CommandLine',
+    'MachO', 'Error', 'StringSaver', 'thread', 'TargetParser',
+    'ARMBuildAttributes', 'ARMTargetParser', 'AArch64TargetParser']
```

Comment:
What we really needed was
- lib/Support/Windows/Host.inc
- lib/Support/Windows/Memory.inc
- lib/Support/Windows/Mutex.inc
- lib/Support/Windows/Path.inc
- lib/Support/Windows/Process.inc
- lib/Support/Windows/Signals.inc
- lib/Support/Windows/TimeValue.inc
- lib/Support/Windows/WindowsSupport.h

Once those were added, I went through and fixed the build errors by including the missing fields. Where linker errors occurred, I added the cpp files to the llvmSupport project.

I was reluctant to start knife editing code, as that would make the code very hard for you to review